test-suite/schema-generation/group-as/group-as-array-bounded_metaschema.xml

```diff
@@ -3,15 +3,15 @@
 <!-- OSCAL CATALOG METASCHEMA -->
 <!-- validate with XSD and Schematron (linked) -->
 <METASCHEMA xmlns="http://csrc.nist.gov/ns/oscal/metaschema/1.0">
-   <schema-name>Metaschema Unit Test: group-as</schema-name>
+   <schema-name>Metaschema Unit Test: group-as bounded</schema-name>
    <schema-version>1.0-milestone1</schema-version>
-   <short-name>metaschema-group-as</short-name>
+   <short-name>metaschema-group-as-bounded</short-name>
    <namespace>http://csrc.nist.gov/ns/metaschema/unit-test/group-as-array-bounded</namespace>
    <json-base-uri>http://csrc.nist.gov/ns/metaschema/unit-test/group-as-array-bounded</json-base-uri>
    <define-assembly name="parent">
       <formal-name>parent</formal-name>
       <description>parent assembly</description>
-      <root-name>parent</root-name>
+      <root-name>group-ass-array-bounded-parent</root-name>
```

Comment:
Minor non-blocking: perhaps the future of a fun inside joke, but I assume you want to change the typo. (I personally don't mind it.)

Reply:
```suggestion
      <root-name>group-as-array-bounded-parent</root-name>
```